app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class UserController < ApplicationController
+      # skip_before_action :authenticate_user!, only: %i[index]
+      def index
```

Comment:
I don't think index should be used for this. Usually index method contains code for returning list of objects.

Reply:
oh i gonna delete this cause found no use